.circleci/config.yml

```diff
@@ -78,6 +80,7 @@ workflows:
       - instrumentation-tests:
           requires:
             - prepare-and-assemble
+      - androidauto-test
```

Comment:
An option here, is make it be the `modules-androidtest`. If we wanted to keep certain tests in their modules (for example, add the maneuver bitmap rendering tests to the `libnavui-maneuver/src/androidTest` 🤔